### ISSUE TITLE
Debug Failure. False expression. at doChange

### ISSUE THREAD

**Issue by cevek** [Microsoft/TypeScript#25723]
**TypeScript Version:**  3.0.0-dev.20180712

**Search Terms:**

**Code**

```ts
Error: Debug Failure. False expression.
    at doChange (/usr/local/lib/node_modules/typescript/lib/tsserver.js:106093:22)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:106084:96
    at Function.ChangeTracker.with (/usr/local/lib/node_modules/typescript/lib/tsserver.js:102880:17)
    at Object.getCodeActions (/usr/local/lib/node_modules/typescript/lib/tsserver.js:106084:64)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:103819:121
    at Object.flatMap (/usr/local/lib/node_modules/typescript/lib/tsserver.js:498:25)
    at Object.getFixes (/usr/local/lib/node_modules/typescript/lib/tsserver.js:103819:23)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:111665:35
    at Object.flatMap (/usr/local/lib/node_modules/typescript/lib/tsserver.js:498:25)
    at Object.getCodeFixesAtPosition (/usr/local/lib/node_modules/typescript/lib/tsserver.js:111663:23)
    at IOSession.Session.getCodeFixes (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119225:64)
    at Session.handlers.ts.createMapFromTemplate._a.(anonymous function) (/usr/local/lib/node_modules/typescript/lib/tsserver.js:118099:61)
    at /usr/local/lib/node_modules/typescript/lib/tsserver.js:119389:88
    at IOSession.Session.executeWithRequestId (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119380:28)
    at IOSession.Session.executeCommand (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119389:33)
    at IOSession.Session.onMessage (/usr/local/lib/node_modules/typescript/lib/tsserver.js:119409:35)
    at Interface.<anonymous> (/usr/local/lib/node_modules/typescript/lib/tsserver.js:120676:27)
```

**Expected behavior:**

**Actual behavior:**

**Playground Link:** 

**Related Issues:** 

tsserver log id: 3897983895

**Comment by andy-ms** [comment]
Probably fixed by #25388.

**Comment by andy-ms** [comment]
@cevek Please make a new issue if you see this again.
